push-messaging-and-notifications/README.md

```diff
@@ -5,7 +5,12 @@ Learn more at http://www.chromestatus.com/feature/5416033485586432 and http://ww
 
 To use this sample please do the following:
 
-1. Create a project in the Google Developer Console
-2. Enable GCM support
-3. Replace "<YOUR API KEY HERE>" in main.js with your own API key from the Google Developer Console project
-4. Replace "<YOUR PROJECT NUMBER>" in manifest.json with your own project number from the Google Developer Console project
+1. Create a project in the [Google Developer Console](https://console.developers.google.com) (detailed instructions are availabl at https://developer.android.com/google/gcm/gs.html)
+1. Enable GCM support
+1. Create a copy of [config.sample.js](config.sample.js) called config.js
+1. Create a copy of [manifest.sample.json](manifest.sample.json) called manifest.json
+1. Replace `<Your Public API Key>` in your new `config.js` file with your own API key from the Google Developer Console project
+4. Replace `<Your Project Number>` in your new `manifest.json` with your own project number from the Google Developer Console project
```

Comment:
`<Your Project Number>` → `<Your Project Number ...>`
(Just to emphasize that there's more characters in the string.)